src/main/java/net/sf/jabref/shared/exception/DatabaseNotSupportedException.java

```diff
@@ -0,0 +1,14 @@
+package net.sf.jabref.shared.exception;
+
+import net.sf.jabref.shared.DBMSProcessor;
+
+/**
+ * This exception is thrown in case that the shared database is not compatible with the current shared database support mechanisms.
+ * See JavaDoc of the field <code>VERSION</code> in {@link DBMSProcessor}.
+ */
+public class DatabaseNotSupportedException extends Exception {
```

Comment:
The exception name does not match the text. Why is not "is not supported" used as text and the "longer" ommitted?